src/Json.Schema.ToDotNet/DataModelGeneratorSettings.cs

```diff
@@ -82,6 +82,12 @@ public class DataModelGeneratorSettings
         /// </summary>
         public bool GenerateCloningCode { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating what C# type to generate for Json type
```

Comment:
>Gets or sets a value indicating what C# type to generate for Json type [](http://example.com/codeflow?start=12&length=70)

Use this:

Gets or sets a value indicating what C# type to emit for Json integers.

Reply:
Updated.